### PR TITLE
Post 2016 changes

### PR DESCRIPTION
**Pull request by MarijeP**
https://github.com/scaleconf/scaleconf.github.com/issues/225
https://github.com/scaleconf/scaleconf.github.com/issues/221
https://github.com/scaleconf/scaleconf.github.com/issues/219

@adrianmoisey do you want to do the issues you’ve assigned to yourself and add to this branch?

### CODE REVIEW

**Comment by adrianmoisey** [comment]
:+1: yup

**Comment by imsickofmaps** [comment]
@adrianmoisey @MarijeP can someone check my last commits?

File: contact/index.html [hunk 1]
```diff
@@ -27,5 +27,8 @@ <h2 id="general">General</h2>
                     </p>
                 </p>
 
+                <h2>Facebook</h2>
+                <p class="details"><a href='http://fb.com/scaleconf'>fb.com/scaleconf</a></p>
```
Comment (adrianmoisey) [inline contact/index.html]: I prefer "www.facebook.com/scaleconf"

**Comment by adrianmoisey** [comment]
@imsickofmaps seems sane.

**Pull request merged**